table-webview/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "table-webview",
+  "version": "0.0.1",
+  "description": "Table as Webview",
+  "author": "Kiel University <rt-kieler-devel@informatik.uni-kiel.de>",
+  "license": "EPL-2.0",
+  "keywords": [
+    "vscode"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kieler",
+    "directory": "tableWebview"
+},
+  "files": [
+    "out",
+    "src",
+    "css"
+  ],
+  "dependencies": {
+    "reflect-metadata": "^0.1.13"
+  },
+  "devDependencies": {
+    "css-loader": "^6.5.1",
+    "file-loader": "^6.2.0",
+    "rimraf": "latest",
```

Comment:
I think the community tries to always have a set minimum version instead of just a latest tag.